api/src/main/java/org/apache/iceberg/metrics/IntCounter.java

```diff
@@ -55,6 +73,6 @@ public Integer value() {
 
   @Override
   public String toString() {
-    return MoreObjects.toStringHelper(this).add("counter", counter).toString();
+    return value().toString();
```

Comment:
Minor: I think it would be strange to print this and see just the value, but it's probably okay.

Reply:
this is superseded now by https://github.com/apache/iceberg/pull/5386 where the name of the counter is also included